pkg/agent/multicluster/stretched_networkpolicy_controller.go

```diff
@@ -184,7 +184,7 @@ func (s *StretchedNetworkPolicyController) processNextWorkItem() bool {
 
 	if podRef, ok := obj.(types.NamespacedName); !ok {
 		s.queue.Forget(obj)
-		klog.Errorf("Expected type 'NamespacedName' in work queue but got object", "object", obj)
+		klog.Errorf("Expected type 'NamespacedName' in work queue but got object %v\n", obj)
```

Comment:
No need to append "\n", klog.Errorf will handle it.

Reply:
use structured logging (`klog.ErrorS`)